### DISCUSSION TITLE
Api create album always results in status 400 Unable to do that

### DISCUSSION THREAD

**Discussion by thatsalltheyare90** [photoprism/photoprism#4262] [Q&A] [answered]
Hi,
I want to use the API to create albums and put photos in it. I'm using a laravel project for this.
I have created in access token and I'm able to fetch all originals using the photoprism's api (my access token is with full access).
But when I try to make a new album, I always get the following response:
`{"code":400,"error":"Unable to do that"}`

I'm sending this to photoprism:
```
  "headers" => [
    "Authorization" => "Bearer xxxxxx-xxxxxxx-xxxxxxx-xxxxxxx"
    "User-Agent" => "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:125.0) Gecko/20100101 Firefox/125.0"
  ]
  "form_params" => [
    "Title" => "title"
    "Description" => "description"
    "Notes" => ""
    "Favorite" => false
  ]
]
```

I've tried various combinations in the form_params:
```
 'form_params' => [
                    'data' => [
                        'title'       => 'title',
                        'description' => 'description',
                        'notes'       => '',
                        'favorite'    => false,
                    ],
                ],
```

```
        'form_params' => [
                    'data' => json_encode([
                        'title'       => 'title',
                        'description' => 'description',
                        'notes'       => '',
                        'favorite'    => false,
                    ]),
                ],
```

```
'form_params' => [
                    json_encode([
                        'title'       => 'title',
                        'description' => 'description',
                        'notes'       => '',
                        'favorite'    => false,
                    ]),
                ]
```

Also I've tried with capitalizing the keys (Title instead of title), but I always get the same response.
I must be doing something wrong here but I have no idea what that is unfortunately.
I've tried to look ip up in the api documentation, but I don't have any knowledge about go so I couldn't take much from that code.

**Comment by lastzero** [comment, marked as answer]
See:
- https://github.com/photoprism/photoprism/issues/4354
- https://github.com/photoprism/photoprism/issues/2132
